# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1406621885 -7200
#      Tue Jul 29 10:18:05 2014 +0200
# Node ID 69658bd9952b8b5aa6f2f2cbb08e2dff7c6395b3
# Parent  b2db129c664d77c7a011c797d205935a76c845a5
__add_default_menu__.m: Fix bug that findobj is missing HLIST

diff --git a/scripts/plot/util/private/__add_default_menu__.m b/scripts/plot/util/private/__add_default_menu__.m
--- a/scripts/plot/util/private/__add_default_menu__.m
+++ b/scripts/plot/util/private/__add_default_menu__.m
@@ -88,17 +88,17 @@ function __save_as__ (caller)
 endfunction
 
 
 function hax = __get_axes__ (h)
   ## Get parent figure
   fig = ancestor (h, "figure");
 
   ## Find all axes which aren't legends
-  hax = findobj ("type", "axes", "-not", "tag", "legend");
+  hax = findobj (fig, "type", "axes", "-not", "tag", "legend");
 endfunction
 
 function grid_cb (h, e)
   hax = __get_axes__ (h);
   id = get (h, "tag");
   switch (id)
     case "toggle"
       arrayfun (@grid, hax);
